packages/gatsby-plugin-google-gtag/README.md

```diff
@@ -0,0 +1,116 @@
+# gatsby-plugin-google-gtag
+
+Easily add Google Global Site Tag to your Gatsby site.
+
+## Install
+
+`npm install --save gatsby-plugin-google-gtag`
+
+## How to use
+
+```javascript
+// In your gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-plugin-google-gtag`,
+      options: {
+        // You can add multiple tracking ids and a pageview event will be fired for all of them.
+        trackingIds: [
+          'GA-TRACKING_ID', // Google Analytics / GA
+          'AW-CONVERSION_ID', // Google Ads / Adwords / AW
+          'DC-FLOODIGHT_ID', // Marketing Platform advertising products (Display & Video 360, Search Ads 360, and Campaign Manager)
+        ],
+        // This object gets passed directly to the gtag config command
+        // This config will be shared accross all trackingIds
+        gtagConfig: {
+          optimize_id: 'OPT_CONTAINER_ID',
+          anonymize_ip: true,
+        },
+        // This object is used for configuration specific to this plugin
+        pluginConfig: {
+          // Puts tracking script in the head instead of the body
+          head: false,
+          // Setting this parameter is also optional
+          respectDNT: true,
+          // Avoids sending pageview hits from custom paths
+          exclude: ['/preview/**', '/do-not-track/me/too/'],
+        },
+      },
+    },
+  ],
+}
+```
+
+## Custom Events
+
+This plugin automatically sends a "pageview" event to all products given as "trackingIds" on every Gatsbys route change.
+
+If you want to call a custom event you have access to `window.gtag` where you can call an event for all products:
+
+```bash
+window.gtag('event', 'click', { ...data });
+```
+
+or you can target a specific product:
+
+```bash
+window.gtag('event', 'click', { send_to: 'AW-CONVERSION_ID', ...data });
+```
+
+## `<OutboundLink>` component
+
+To make it easy to track clicks on outbound links the plugin provides a component.
```

Comment:
Not sure if it's worth calling this ExternalLink, instead of OutboundLink? Is Outbound a term used by gtag?